tests/sentry/api/endpoints/test_organization_member_team_details.py

```diff
@@ -111,6 +111,189 @@ def test_can_join_as_member_with_open_membership(self):
+        user = self.create_user('dummy@example.com')
+        member_om = self.create_member(
+            organization=organization,
+            user=user,
+            role='manager',
+            teams=[],
+        )
+
+        path = reverse(
+            'sentry-api-0-organization-member-team-details',
+            args=[
+                organization.slug,
+                member_om.id,
+                team.slug,
+            ]
+        )
+
+        self.login_as(owner)
+
+        resp = self.client.post(path)
+
+        assert resp.status_code == 201
+
+        assert OrganizationMemberTeam.objects.filter(
+            team=team,
+            organizationmember=member_om,
+        ).exists()
+
+    def test_owner_can_add_other_owner(self):
+        owner = self.create_user()
+        organization = self.create_organization(name='foo', owner=owner)
+        organization.flags.allow_joinleave = False
+        organization.save()
+        team = self.create_team(name='foo', organization=organization)
+        user = self.create_user('dummy@example.com')
+        member_om = self.create_member(
+            organization=organization,
+            user=user,
+            role='owner',
+            teams=[],
+        )
+
+        path = reverse(
+            'sentry-api-0-organization-member-team-details',
+            args=[
+                organization.slug,
+                member_om.id,
+                team.slug,
+            ]
+        )
+
+        self.login_as(owner)
+
+        resp = self.client.post(path)
+
+        assert resp.status_code == 201
+
+        assert OrganizationMemberTeam.objects.filter(
+            team=team,
+            organizationmember=member_om,
+        ).exists()
+
+    def test_manager_can_add_member(self):
+        manager = self.create_user()
+        organization = self.create_organization(name='foo')
+        team = self.create_team(name='foo', organization=organization)
+        self.create_member(
+            organization=organization,
+            user=manager,
+            role='manager',
+            teams=[team],
+        )
+        organization.flags.allow_joinleave = False
+        organization.save()
+        user = self.create_user('dummy@example.com')
+        member_om = self.create_member(
+            organization=organization,
+            user=user,
+            role='member',
+            teams=[],
+        )
+
+        path = reverse(
+            'sentry-api-0-organization-member-team-details',
+            args=[
+                organization.slug,
+                member_om.id,
+                team.slug,
+            ]
+        )
+
+        self.login_as(manager)
+
+        resp = self.client.post(path)
+
+        assert resp.status_code == 201
+
+        assert OrganizationMemberTeam.objects.filter(
+            team=team,
+            organizationmember=member_om,
+        ).exists()
+
+    def test_manager_cannot_add_owner(self):
```

Comment:
do we want to enforce this specific part? seems harmless

Reply:
i just wanted to keep it as close to the current permissions as possible for now